graphics/oofcanvas/Portfile

```diff
@@ -0,0 +1,60 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           python 1.0
+
+name                oofcanvas
+version             1.0.1
+revision            0
+
+license             public-domain
+categories          graphics
+platforms           darwin
+
+maintainers         {@snarkhunter nist.gov:stephen.langer} \
+                    nist.gov:oof_manager
+
+description         C++ library that does some of what libgnomecanvas used to do
+
+long_description    OOFCanvas is a replacement for libgnomecanvas, designed \
+                    for use in OOF2, but hopefully useful elsewhere. OOFCanvas \
+                    is based on Cairo and, unlike libgnomecanvas, is compatible\
+                    with gtk3. The canvas is a drawing area that can display a \
+                    variety of shapes, including text. It can be scrolled, \
+                    zoomed, and printed.  Items drawn on the canvas can be \
+                    selected with the mouse. Note that OOFCanvas is not a \
+                    drop-in replacement for libgnomecanvas.  It's also not a \
+                    full-fledged gtk widget.  It's a set of classes that does \
+                    some of what libgnomecanvas did and uses gtk.
+
+homepage            https://www.ctcms.nist.gov/oof/oofcanvas
+master_sites        https://www.ctcms.nist.gov/oof/oofcanvas/source
+distfiles           oofcanvas-${version}
+
+checksums           rmd160 1868451056ad15636f1a03cc165e0296ca58bf28 \
+                    sha256 15760ce9c8bda341e6160e55899ed6bc8056a5fd1f9b1fb68f64fe9c0a3700b4 \
+                    size 1111805
+
+python.default_version  27
+
+destroot.args        --prefix=${prefix} --port-dir=${prefix} --skip-build
+
+depends_build       port:pkgconfig
+
+depends_lib-append  port:gtk3 \
+                    port:cairomm \
+                    port:adwaita-icon-theme
+
+variant pythonAPI description {Build the python API} {
+    build.args-append       --pythonAPI
+    destroot.args-append    --pythonAPI
+    depends_lib-append      port:py27-gobject3
+}
+
+variant magick description {Include code to display ImageMagick images} {
+    build.args-append       --magick
+    destroot.args-append    --magick
+    depends_lib-append      port:imagemagick
+}
+
+default_variants    +pythonAPI +magick
```

Comment:
is this port useful at all without having these variants enabled? If not, then I guess there is no need for the variants and you can just fold this into the main port